src/Hosting/test/FunctionalTests/LinkedApplicationTests.cs

```diff
@@ -41,8 +41,6 @@ public async Task LinkedApplicationWorks()
                     StatusMessagesEnabled = false
                 };
 
-                deploymentParameters.EnvironmentVariables["COREHOST_TRACE"] = "1";
```

Comment:
Why?

Reply:
I believe that was just added to help debug the CI issues: https://github.com/dotnet/aspnetcore/pull/33187/commits/bb2e26309c069a00756f7fc92f8876d067a8aee5